### PR TITLE
Fix Content-Type header

### LINKED ISSUE #19: Sample code error found .
I simplely run the sample code  like this .

```golang
package main

import (
	"fmt"

	skynet "github.com/NebulousLabs/go-skynet"
)

func main() {
	// upload
	skylink, err := skynet.UploadFile("./src.jpg", skynet.DefaultUploadOptions)
	if err != nil {
		fmt.Printf("Unable to upload: %v", err.Error())
		return
	}
	fmt.Printf("Upload successful, skylink: %v\n", skylink)

	// download
	err = skynet.DownloadFile("./dst.jpg", skylink, skynet.DefaultDownloadOptions)
	if err != nil {
		fmt.Printf("Something went wrong, please try again.\nError: %v", err.Error())
		return
	}
	fmt.Println("Download successful")
}

```

But after , build and run , one error happened. Any one can tell me why . 

Error Content : 

```shell
Unable to upload: [could not execute request; error code received; 400 response from POST: failed parsing Content-Type header: mime: no media type; error response]
```

### PR DESCRIPTION
**Pull request by mrcnski**
Fixes #19

### CODE REVIEW

File: utils_test.go [hunk 1]
```diff
@@ -1,9 +1,65 @@
...
+	"net/url"
 	"testing"
 )
 
+const (
+	portalURL = DefaultPortalURL
+)
+
+// TestMakeURL tests making URLs.
+func TestMakeURL(t *testing.T) {
```
Comment (MSevey) [inline utils_test.go:13]: nit: might be worth making these test cases be the actual API endpoints. Could then be extended every time we add an endpoint.  

The benefit I see is that then we also are explicitly testing the formation of all the urls for the SDK.

**Review by MSevey — APPROVED** [review]

File: utils_test.go (hunk 1 above)
Comment (mrcnski) [reply utils_test.go:13]: All the endpoints should be tested by the mock tests, which try to match requests to the expected full URLs. That should be enough to ensure that `makeURL` is working for all the default endpoints. My main concern and reason for this unit test is that people can pass in their own custom endpoint paths, for that I think we just need to test the possible edge cases concerning slashes
